bot/cogs/jams.py

```diff
@@ -24,11 +24,22 @@ def __init__(self, bot: commands.Bot):
         team_name: str, members: commands.Greedy[Member]
     ):
         """
-        Create a team channel in the Code Jams category, assign roles and then add
-        overwrites for the team.
+        Create a team channel (both voice and text) in the Code Jams category, assign roles
+        and then add overwrites for the team.
 
         The first user passed will always be the team leader.
         """
+
+        # We had a little issue during Code Jam 4 here, the greedy converter did it's job
+        # and ignored anything which wasn't a valid argument which left us with teams of
+        # two members or at some times even 1 member. This fixes that by checking that there
+        # are always 3 members in the members list.
+        if len(members) < 3:
+            await ctx.send(f":no_entry_sign: One of your arguments was invalid - there must be a "
+                           f"minimum of 3 valid members in your team. Found: {len(members)} "
+                           f"members")
```

Comment:
Does this have to be an f-string?

Reply:
yes. otherwise flake8 will error and I will have to make it look like
```py
blah(f"abc {d}"
        "world")
```